core/driver/sources/src/main/scala/org/virtuslab/ideprobe/ide/intellij/InstalledIntelliJ.scala

```diff
@@ -3,16 +3,18 @@ package org.virtuslab.ideprobe.ide.intellij
 import java.io.File
 import java.net.ServerSocket
 import java.nio.ByteBuffer
-import java.nio.file.{Path, Paths}
-import com.typesafe.config.ConfigRenderOptions
+import java.nio.file.{Files, Path, StandardCopyOption}
 import com.zaxxer.nuprocess.{NuAbstractProcessHandler, NuProcessBuilder}
 import org.virtuslab.ideprobe.Extensions._
 import org.virtuslab.ideprobe._
 import org.virtuslab.ideprobe.config.DriverConfig
 import org.virtuslab.ideprobe.jsonrpc.JsonRpcConnection
+
 import scala.concurrent.{ExecutionContext, blocking}
 
-final class InstalledIntelliJ(val root: Path, probePaths: IdeProbePaths, config: DriverConfig) {
+sealed abstract class InstalledIntelliJ(root: Path, probePaths: IdeProbePaths, config: DriverConfig) {
+  def cleanup(): Unit
+
   val paths: IntelliJPaths = new IntelliJPaths(root, config.headless)
```

Comment:
Existing IntelliJ instance can have specific directory config (specified in `*.properties`). We want those paths to be manually configurable. (or maybe we can automatically detect the existing directory structure, for example by reading those locations from existing `*.properties` or by providing default structure?)

Reply:
Done.